web3/_utils/request.py

```diff
@@ -48,6 +47,9 @@ def cache(self, key: str, value: Any) -> Dict[str, Any]:
     def get_cache_entry(self, key: str) -> Any:
         return self._data[key]
 
+    def clear(self) -> None:
```

Comment:
What was the reason this was added in? Is this used anywhere?

Reply:
It is used in test_request.py for clearing the cache during testing.

Reply:
Ah, yep I see it, ty.